### PR TITLE
Add support for location_type filter & readme update

### PR DESCRIPTION
**Pull request by nadesh-git**
- Updating lib for location_type support
- Updating README.md

### CODE REVIEW

**Comment by shukerullah** [comment]
Hey, thanks for the PR. Can you also update the lib folder?

**Comment by nadesh-git** [comment]
> Hey, thanks for the PR. Can you also update the lib folder?

Hi, sure. Do you have command to generate the lib folder?

File: src/index.js [hunk 1]
```diff
@@ -82,6 +82,17 @@ export default {
...
   },
 
+  /**
+   *
+   *
+   * @param {string} region
+   * Accepted values: ROOFTOP, RANGE_INTERPOLATED, GEOMETRIC_CENTER, APPROXIMATE
+   */
+  setLocationType(locationType) {
+    LOCATIONTYPE = locationType;
```
Comment (shukerullah) [inline src/index.js]: `LOCATIONTYPE` is not defined in the scope.

Comment (nadesh-git) [reply src/index.js]: Pushed

**Comment by shukerullah** [comment]
@nadesh-git can you allow edits from maintainers? Thanks

[Allowing changes to a pull request branch created from a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

File: src/index.js [hunk 2]
```diff
@@ -82,6 +82,17 @@ export default {
     DEBUG = flag;
   },
 
+  /**
+   *
+   *
+   * @param {string} region
```
Comment (shukerullah) [inline src/index.js]: Please update doc. Write locationType instead region.

**Comment by nadesh-git** [comment]
> @nadesh-git can you allow edits from maintainers? Thanks
> 
> [Allowing changes to a pull request branch created from a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

Already enabled the Allow edits by maintainers

File: src/index.js (hunk 2 above)
Comment (nadesh-git) [reply src/index.js]: Updated now

**Comment by shukerullah** [comment]
@nadesh-git I have updated PR, please verify and let me know. I will merge and update on npm.

**Comment by nadesh-git** [comment]
> @nadesh-git I have updated PR, please verify and let me know. I will merge and update on npm.

Everything fine! Great work!

**Pull request merged**